src/containers/DataDisplay/DataViewTable.tsx

```diff
@@ -96,11 +186,17 @@ const getCellComponent = (value: any, columnDetails: HeaderRow, id: number) => {
 const getAPIQueryParams = ({ page, perPage, sortBy }: any) => {
   const offset = page && perPage ? String((Number(page) - 1) * Number(perPage)) : undefined
   let orderBy: string | undefined = undefined
-  let ascending: string | undefined = undefined
+  let ascending: 'true' | 'false' | undefined = undefined
```

Comment:
Not sure why you needed to do that instead of `let ascending: 'true' | 'false' = 'false'` And also why these need to be string instead of boolean

Reply:
> Not sure why you needed to do that instead of let ascending: 'true' | 'false' = 'false'

Because we don't want anything to show in the url when it's default (i.e. true)

> And also why these need to be string instead of boolean

Cos they're url query parameters, not booleans. ;)

Reply:
Hmmm I see! But wouldn't this only show IF you are sorting a column?

Reply:
Yes, sorry -- the undefined is for when it's not sorted at all (i.e. default ordering --  usually by id)